src/utils/common.js

```diff
@@ -32,3 +32,5 @@ export const formatPhoneNumber = phoneNumber => {
 
   return `${countryCode} (${areaCode}) ${firstPart} ${secondPart}`;
 };
+
+export const capitalizeFirstLetter = inputString => inputString.charAt(0).toUpperCase() + inputString.slice(1);
```

Comment:
@Yivgha let's rename it to just `capitalize` it's more convenient, many utility libs has this name for such function